src/script/components/userList.ts

```diff
@@ -178,17 +181,32 @@ ko.components.register('user-list', {
 
     this.cssClasses = ko.pureComputed(() => (isCompactMode ? 'search-list-sm' : 'search-list-lg'));
 
+    this.onUserKeyPressed = (userEntity: User, event: KeyboardEvent) => {
+      if (isSpaceKey(event) || isEnterKey(event)) {
+        onClickOrKeyPressed(userEntity, event);
+      }
+      return true;
+    };
+
     this.onUserClick = (userEntity: User, event: MouseEvent) => {
+      onClickOrKeyPressed(userEntity, event);
```

Comment:
can't the `onClickOrKeyPressed` event handler be used directly instead of using `onUserClick`?

Reply:
but then you will have to detect if it's a `MouseEvent` or a `KeyboardEvent` to avoid TS errors and the final code doesn't look cleaner